src/app/shared/components/fy-select/fy-select-modal/fy-select-modal.component.ts

```
@@ -92,14 +83,14 @@
     }
   }
 
-  ngAfterViewInit() {
+  ngAfterViewInit(): void {
     if (this.searchBarRef && this.searchBarRef.nativeElement) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)